identity/src/commonMain/kotlin/com/android/identity/device/DeviceCheck.kt

```diff
@@ -0,0 +1,39 @@
+package com.android.identity.device
+
+import com.android.identity.securearea.SecureArea
+import kotlinx.io.bytestring.ByteString
+
+/**
+ * Generates statements validating device/OS/app integrity. Details of these
+ * statements are inherently platform-specific.
+ */
+expect object DeviceCheck {
```

Comment:
Since this can only be implemented on iOS and Android should we introduce a `uiMain` sourceSet? That way we avoid a dummy `DeviceCheck.jvm.kt` file and like also `DeviceAttestationJvm.kt`...

(I'm already contemplating doing this for other things, see the [mp-nfc-engagement branch](https://github.com/openwallet-foundation-labs/identity-credential/blob/mp-nfc-engagement/identity/build.gradle.kts#L80)...)

Reply:
Or maybe a better name is `mobileOsMain` (naming is hard, news at 11)